### PR TITLE
Added Footer To Dashboard

### PR DESCRIPTION
**Pull request by NavyAdmiral**
[Addresses #158877959](https://www.pivotaltracker.com/story/show/158877959)

This is how it looks like:
image

I don't like how I've handled static.config.js. Also, I don't think footer should be displayed on every page. Displaying it on every page leads to displaying it on the About page. This in turn would lead to showing the same information twice, which wouldn't be too good.

### CODE REVIEW

**Comment by thodges-gh** [comment]
This is great, thank you! I agree about not displaying the footer on every page.

File: gui/src/containers/Jobs.js [hunk 1]
```diff
@@ -83,6 +84,7 @@ export class Jobs extends Component {
             {renderSidebar(this.props)}
           </Grid>
         </Grid>
+          <Footer/>
```
Comment (rupurt) [inline gui/src/containers/Jobs.js]: <- `\t`

File: gui/src/components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
```
Comment (rupurt) [inline gui/src/components/Footer.js]: ✂️ unused `Component, Grid, classNames` imports

File: gui/src/components/Footer.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import { withStyles } from "@material-ui/core/styles";
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
+import classNames from "classnames";
+import { withRouteData } from "react-static";
+
+const styles = theme => ({
+  style: {
+    backgroundColor: "#F8F8F8",
```
Comment (rupurt) [inline gui/src/components/Footer.js]: I feel like these values for color & spacing should come from the MaterialUI theme. There is some example usage [here](https://github.com/smartcontractkit/chainlink/blob/master/gui/src/components/PaddedCard.js#L8) and if you'd like to read more about how it works there is a bunch of information on the [MaterialUI website](https://material-ui.com/customization/themes/#themes)

File: gui/src/components/Footer.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    display: "block",
+    padding: "20px",
+    height: "60px",
+    width: "100%"
+  }
+});
+
+const Footnote = ({ classes, version, sha }) => {
+  return (
+    <div className={classes.wrapper}>
```
Comment (rupurt) [inline gui/src/components/Footer.js]: Can this use the [Card](https://material-ui.com/api/card/) component?

File: gui/static.config.js [hunk 5]
```diff
@@ -4,26 +4,32 @@ import JssProvider from 'react-jss/lib/JssProvider'
 import { MuiThemeProvider, createMuiTheme, createGenerateClassName } from '@material-ui/core/styles'
 import theme from './src/theme' // Custom Material UI theme
 
+const getVersion = () => {
```
Comment (rupurt) [inline gui/static.config.js]: How do you feel about calculating this once and storing it in a variable? We could then return the variable from the function e.g.

```javascript
// ...
const buildInfo = extractBuildInfo()

// ...
getData: () => buildInfo
```

Comment (NavyAdmiral) [reply gui/static.config.js]: Yea this is what I felt was wrong initially. Now I refactored the function for buildInfo in utils; take a look at the commit below.

File: gui/src/components/Footer.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+import Card from '@material-ui/core/Card'
+
+const styles = theme => ({
+  style: {
+    textAlign: "center",
+    padding: "20px",
+    position: "fixed",
+    left: "0",
+    bottom: "0",
+    height: "60px",
```
[the remark this answers is not in the document]
Comment (NavyAdmiral) [reply gui/src/components/Footer.js]: height was extra but I'm pretty sure pos, left and bottom should be there

**Pull request merged**